src/pages/Environments.js

```diff
@@ -704,7 +704,7 @@ export const Environments = ({ nav = undefined }) => {
                 const cloudEnvironment = filteredCloudEnvironments[rowIndex]
                 const computeType = isApp(cloudEnvironment) ? 'app' : 'runtime'
                 return h(Fragment, [
-                  renderPauseButton(computeType, cloudEnvironment),
+                  renderPauseButton(computeType, cloudEnvironment, getCloudProvider(filteredCloudEnvironments[rowIndex]) === 'AZURE_VM'),
```

Comment:
lets compute if its azure in the function itself, since we already pass the `cloudEnvironment`